pkg/lib/cockpit-components-dropdown.tsx

```diff
@@ -36,7 +35,14 @@ import { EllipsisVIcon } from '@patternfly/react-icons';
  * require a separator between DropdownItem's use PatternFly's Divivder
  * component.
  */
-export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props }) => {
+export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props } : {
+    dropdownItems: React.ReactNode,
+    position?: "start" | "end" | "right" | "left" | "center",
```

Comment:
Can't we import the definitions from PatternFly here? Its `DropdownPopperProps.position` 

https://www.patternfly.org/components/menus/select/#selectpopperprops

Reply:
Yes.  That works and is indeed nicer / DRY.